src/middlewared/middlewared/plugins/truenas_connect/update.py

```diff
@@ -135,7 +129,7 @@ async def do_update(self, data):
             'ips': data['ips'],
             'interfaces': data.get('interfaces', []),
```

Comment:
```suggestion
            'interfaces': data['interfaces'],
```
Should be guaranteed in `self.config()` right?